File: test-image.js
const fs = require('fs');
const { createCanvas } = require('canvas');

// Create a 200x200 canvas with a red circle
const width = 200;
const height = 200;
const canvas = createCanvas(width, height);
const ctx = canvas.getContext('2d');

// Fill the background
ctx.fillStyle = 'white';
ctx.fillRect(0, 0, width, height);

// Draw a red circle
ctx.fillStyle = 'red';
ctx.beginPath();
ctx.arc(width/2, height/2, 50, 0, Math.PI * 2);
ctx.fill();

// Save the image
const buffer = canvas.toBuffer('image/png');
fs.writeFileSync('test-image.png', buffer);

console.log('Test image created: test-image.png');

File: test_image.py
import numpy as np
from PIL import Image

# Create a simple test image
width, height = 200, 200
image = np.zeros((height, width, 3), dtype=np.uint8)

# Create a red circle in the center
center_x, center_y = width // 2, height // 2
radius = 50
for y in range(height):
    for x in range(width):
        if (x - center_x) ** 2 + (y - center_y) ** 2 < radius ** 2:
            image[y, x] = [255, 0, 0]  # Red

# Save the image
img = Image.fromarray(image)
img.save('test_image.png')

print("Test image created: test_image.png")
